src/lib/features/instance-stats/instance-stats-service.ts

```diff
@@ -58,6 +59,8 @@ export interface InstanceStats {
     strategies: number;
     SAMLenabled: boolean;
     OIDCenabled: boolean;
+    passwordAuthEnabled: boolean;
```

Comment:
Why this change? I can't see where these are consumed, only returned. Were these missing before this?

Reply:
It's returned in line 401. It was previously added to getStats() as a calculation but never returned and never used. The thing is, getStats is only used to sign our stats that we require from our self-hosted customers, so I thought it made sense to have it there